vms/registration/password_validate.py

```diff
@@ -0,0 +1,21 @@
+import re
+
+def validate_password(password):
+	upp_low_spec = False
+	upp_low_num = False
+	upp_num_spec = False
+	low_num_spec = False
+	if re.search(r'[A-Z]',password) and re.search(r'[a-z]',password) and re.search(r'[@#$%^&+=]',password):
+		upp_low_spec = True
+	if re.search(r'[A-Z]',password) and re.search(r'[a-z]',password) and re.search(r'[0-9]',password):
+		upp_low_num = True
+	if re.search(r'[A-Z]',password) and re.search(r'[0-9]',password) and re.search(r'[@#$%^&+=]',password):
+		upp_num_spec = True
+	if re.search(r'[a-z]',password) and re.search(r'[0-9]',password) and re.search(r'[@#$%^&+=]',password):
```

Comment:
redundant regex checks in this function. Change it to avoid wasting computations.